twarep/electrifly
# Download Functionality and Various Bug Fixes

## Checklist

## What I Did
- Changed the global activity bug. Now the flight planning will be session based.
- Got the selector to be in order of descending flight dates
- Took out the TBD and NA from the statistical insights.
- Added units to the custom graph axis titles.
- Added the download feature for the data aggregation.

## How I Did it?
- I used pandas and NumPy manipulation to get the data to merge together.

## How to Test?
- Manual Test. Please make sure that the above mentioned changes work. 
- In all date selectors on the app, see if the dates are in descending order.
- When hosted, see if two people's changes to the flight exercise planning don't overlap.
- See if TBD and NA are in the selectors for the statistical insights.
- See if all the units show for the custom graphs.
- Make sure that the data is downloading and showing as intended when you click "Add Filter" on the Data Preview page.

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
